### PR TITLE
feat: markdown file reader

### PR DESCRIPTION
**Pull request by Aminejvm**
markdown

Currently supports the following element ( headings, paragraph, image, link, lists )

### CODE REVIEW

File: components/core/MarkdownFrame.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import { css } from "@emotion/react";
```
Comment (jimmylee) [inline components/core/MarkdownFrame.js:2]: Please reorder imports to match other files.

File: components/core/MarkdownFrame.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  width: 100%;
+  height: 100%;
+  overflow-y: scroll;
+  will-change: transform;
+  color: ${Constants.system.white};
+`;
+
+const STYLES_BODY = css`
+  width: 100%;
+  max-width: 687px;
```
Comment (jimmylee) [inline components/core/MarkdownFrame.js:21]: Where does `687px` come from?

**Review by jimmylee — APPROVED** [review]
LGTM, just some questions and things you might want to polish and be more intentional about. Slate seems to be designed on a pt4 system so make sure measurements are multiples of 4 or 8 or 12

File: components/core/MarkdownFrame.js (hunk 2 above)
Comment (Aminejvm) [reply components/core/MarkdownFrame.js:21]: I used it as a way to ensure we have a maximum of 70 characters in one line

File: components/core/MarkdownFrame.js (hunk 1 above)
Comment (Aminejvm) [reply components/core/MarkdownFrame.js:2]: I'll make sure to do that

**Pull request merged**